acvm-repo/acir/src/parser/tests.rs

```diff
@@ -331,6 +331,27 @@ fn keccakf1600() {
     assert_circuit_roundtrip(&src);
 }
 
+#[test]
+#[should_panic]
+fn keccakf1600_missing_inputs() {
+    let input_witnesses: Vec<String> = (0..24).map(|i| format!("(_{i}, 64)")).collect();
+    let inputs_str = input_witnesses.join(", ");
+
+    let output_witnesses: Vec<String> = (24..49).map(|i| format!("_{i}")).collect();
+    let outputs_str = output_witnesses.join(", ");
+
+    let src = format!(
+        "
+        current witness index : _49
+        private parameters indices : []
+        public parameters indices : []
+        return value indices : []
+        BLACKBOX::KECCAKF1600 [{inputs_str}] [{outputs_str}]
+        "
+    );
+    assert_circuit_roundtrip(&src);
```

Comment:
Should the failing tests do `Circuit::from_str(src).unwrap()` instead? Mainly for clarity.

Reply:
Sure I will switch to that